astrokat/test/test_nd_timings.py

```diff
@@ -64,7 +64,7 @@ def test_nd_pattern_plus_off(self):
 
         result = LoggedTelescope.user_logger_stream.getvalue()
         self.assertIn("No ND for target", result)
-        self.assertIn("noise-diode off at 1573714914.0", result)
+        self.assertIn("noise-diode off at 1573715222.0", result)
```

Comment:
Making this as a general comment: originally, I mentioned the scan duration relating to the reference scan because the log line the test seemed to be looking for had a different value to what the scan was configured for in its yaml file. I see in the newer commits that there are now changes to other tests as well which I assume are not reference scan tests or use the reference scan yaml. Were these tests failing before? There seems to be some discrepancy in the log line checks.